lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -275,29 +281,16 @@ sub _show {
             scenario                => $job->scenario_name,
             worker                  => $worker,
             assigned_worker         => $job->assigned_worker,
+            resultfiles             => $resultfiles,
+            ulogs                   => $ulogs,
+            show_details            => ($test_modules || grep /autoinst-log.txt/, @$resultfiles),
```

Comment:
Lets avoid the expression form  of `grep` and use blocks instead.

Reply:
I left that part out for another PR and will change it